### PR TITLE
declare types

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: lib/Service.php [hunk 1]
```diff
@@ -252,7 +255,7 @@ public function mapValueObject(string $elementName, string $className)
             return \Sabre\Xml\Deserializer\valueObject($reader, $className, $namespace);
         };
         $this->classMap[$className] = function (Writer $writer, $valueObject) use ($namespace) {
-            return \Sabre\Xml\Serializer\valueObject($writer, $valueObject, $namespace);
```
Comment (staabm) [inline lib/Service.php:255]: a serializer does not return a vlaue

File: lib/Writer.php [hunk 2]
```diff
@@ -182,6 +184,7 @@ public function startElement($name): bool
      * XMLWriter::startElement doesn't either.
      *
      * @param array|string|object|null $content
+     * @param mixed $content
```
Comment (phil-davis) [inline lib/Writer.php:187]: `$content` is mentioned twice ??

**Review by phil-davis — COMMENTED** [review]
and cs-fixer is complaining about a few minor things

File: lib/Writer.php (hunk 2 above)
Comment (staabm) [reply lib/Writer.php]: good catch

**Review by phil-davis — APPROVED** [review]
LGTM

**Comment by staabm** [comment]
thank you phil.

**Pull request merged**